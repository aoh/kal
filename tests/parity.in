every day on odd weeks: odd always
every Wednesday on even weeks: even Wednesday
