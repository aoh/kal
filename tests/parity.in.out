1.1.1970, Thursday, week 1
 - odd always

2.1.1970, Friday, week 1
 - odd always

3.1.1970, Saturday, week 1
 - odd always

4.1.1970, Sunday, week 1
 - odd always

7.1.1970, Wednesday, week 2
 - even Wednesday

12.1.1970, Monday, week 3
 - odd always

13.1.1970, Tuesday, week 3
 - odd always

14.1.1970, Wednesday, week 3
 - odd always

15.1.1970, Thursday, week 3
 - odd always

every day in odd weeks: odd always
every Wednesday in even weeks: even Wednesday
